### ISSUE TITLE
Broken link to template file for assignment-2-worksheet

### ISSUE THREAD

**Issue by gturco** [jnmaloof/BIS180L_web#15]
Answer the questions in the assignment2-worksheet.md file template available on the website

**Comment by jnmaloof** [comment]
yes I know, it is no my to-do.  Probably won't happen until tomorrow

**Comment by jnmaloof** [comment]
(Need to roll it back to some version of 2015 rather than Ian's from 2016)
